releases/release-1.15/release-notes-draft.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1.15 What’s New
 
-A complete changelog for the release notes is now hosted in a customizable format at [https://relnotes.k8s.io/](https://relnotes.k8s.io/). Check it out and please give us your feedback!
+A complete changelog for the release notes is now hosted in a customizable format at [https://relnotes.k8s.io/](https://relnotes.k8s.io/?releaseVersions=1.15.0). Check it out and please give us your feedback!
```

Comment:
```suggestion
A complete changelog for the release notes is now hosted in a customizable format at [https://relnotes.k8s.io/](https://relnotes.k8s.io/?releaseVersions=1.15). Check it out and please give us your feedback!
```

Reply:
I see, compared with https://github.com/kubernetes-sigs/release-notes/pull/63 this makes totally sense. Sorry for the interruption